### PR TITLE
Test out eslint errors and py3k linter

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

File: somefile.py [hunk 1]
```diff
@@ -42,3 +42,8 @@ def run_command(
 def do_thing():
     print 'thinger'
     print 'thing 2'
+
+def things():
+    print 'thinger'
```
Comment (stickler-ci-dev) [inline somefile.py]: E1601 print statement used

File: somefile.py [hunk 2]
```diff
@@ -42,3 +42,8 @@ def run_command(
...
+    from urlparse import urlparse
```
Comment (stickler-ci-dev) [inline somefile.py]: W1648 Module moved in Python 3

File: somefile.py [hunk 3]
```diff
@@ -42,3 +42,8 @@ def run_command(
 def do_thing():
     print 'thinger'
     print 'thing 2'
+
+def things():
```
Comment (stickler-ci-dev) [inline somefile.py]: E302 expected 2 blank lines, found 1

File: somefile.py (hunk 1 above)
Comment (stickler-ci-dev) [inline somefile.py]: E1601 print statement used

File: somefile.py (hunk 2 above)
Comment (stickler-ci-dev) [inline somefile.py]: W1648 Module moved in Python 3

File: somefile.py (hunk 3 above)
Comment (stickler-ci-dev) [inline somefile.py]: E302 expected 2 blank lines, found 1

File: somefile.py (hunk 1 above)
Comment (stickler-ci-dev) [inline somefile.py]: E1601 print statement used

File: somefile.py (hunk 2 above)
Comment (stickler-ci-dev) [inline somefile.py]: W1648 Module moved in Python 3

File: somefile.py (hunk 3 above)
Comment (stickler-ci-dev) [inline somefile.py]: E302 expected 2 blank lines, found 1

File: somefile.py (hunk 1 above)
Comment (markstory) [inline somefile.py]: E1601 print statement used

File: somefile.py (hunk 2 above)
Comment (markstory) [inline somefile.py]: W1648 Module moved in Python 3

File: somefile.py (hunk 3 above)
Comment (markstory) [inline somefile.py]: E302 expected 2 blank lines, found 1

**Pull request closed without merging**